k8s-install/scripts/install-cni.sh

```diff
@@ -114,6 +117,12 @@ if [ -f "/var/run/secrets/kubernetes.io/serviceaccount/token" ]; then
     echo "KUBERNETES_SERVICE_PORT not set"; exit 1;
   fi
 
+  if [ "$SKIP_TLS_VERIFY" == "true" ]; then
```

Comment:
FYI, as 'true' and 'false' are commands, you could simplify this to
```
  if $SKIP_TLS_VERIFY; then
```
OTOH, you may still prefer it as it is.

Reply:
I think what is currently there would have better behavior if something other than `true` or `false` was entered, as something else would probably cause a failed command.